anesthetic/samples.py

```diff
@@ -175,9 +176,21 @@ def plot(self, ax, paramname_x, paramname_y=None, *args, **kwargs):
                                        weights=self.weights,
                                        ncompress=ncompress,
                                        *args, **kwargs)
+                elif plot_type == 'filledkde':
+                    if ncompress is None:
+                        ncompress = 1000
+                    return kde_plot_1d(ax, self[paramname_x],
+                                       weights=self.weights,
+                                       ncompress=ncompress,
+                                       facecolor=True,
+                                       *args, **kwargs)
```

Comment:
Do we need this `plot_type` any more, given that the only difference between `'filledkde'` and `'kde'` is that the argument facecolor=True is passed?

Reply:
No I don't think so. It would be better to remove it I think and then users can just pass facecolor=True.